### ISSUE TITLE
`@secretlint/quick-start` `Error: Not found target files` in version 7.0.0

### ISSUE THREAD

**Issue by uekkie** [secretlint/secretlint#554]
The `config` directory was not included in the package when installing secretlint/quick-start. In version 7

**What version of secretlint are you using?**

7.0.0

**What did you do? Please include the actual steps causing the issue.**

run command
`@secretlint/quick-start`

```
Error: Not found target files
    at runSecretLint (file:///Users/user/.npm/_npx/3efd47c6528c435b/node_modules/secretlint/module/index.js:10:15)
    at async file:///Users/user/.npm/_npx/3efd47c6528c435b/node_modules/@secretlint/quick-start/bin/quick-start.js:5:44
```

result
```
.
├── LICENSE
├── README.md
├── bin
│   └── quick-start.js
├── module
│   ├── index.d.ts
│   ├── index.d.ts.map
│   ├── index.js
│   └── index.js.map
├── package.json
└── src
    └── index.ts
```

**Comment by azu** [comment]
Thanks for report.

> The `config` directory was not included in the package when installing secretlint/quick-start. 

Yes, It is a bug.
https://github.com/secretlint/secretlint/blob/daf5d8ae94344dc8f4a00e1d8742b22d461c97c7/packages/%40secretlint/quick-start/package.json#L38-L41

**Comment by azu** [comment]
Thanks!

Fixed in v7.0.1
https://github.com/secretlint/secretlint/releases/tag/v7.0.1
```
npx @secretlint/quick-start@latest "**/*"
```
